docs/examples.md

```diff
@@ -76,6 +76,26 @@ $y$j9T$A0Y3wwVOKP69S.1K/zYGN.$S596l11UGH3XjN...
 
 The `yescrypt` hashing method is recommended for new passwords. For more details on hashing methods, see `man 5 crypt`.
 
+In case inline definitions of SSH keys are undesirable you can use file references instead. This example creates three users (`core`, `jlebon`, `miabbott`) and sets up their respective SSH keys via file references instead of inlining them. This requires using `butane` with the `--files-dir` parameter to allow loading files from disk.
+
+<!-- butane config -->
+```yaml
+variant: fcos
+version: 1.5.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys_local:
+        - users/core/id_rsa.pub
+    - name: jlebon
+      ssh_authorized_keys_local:
+        - users/jlebon/id_rsa.pub
+        - users/jlebon/id_ed25519.pub
+    - name: miabbott
+      ssh_authorized_keys_local:
+        - users/miabbott/id_rsa.pub
```

Comment:
The extra users complicate the example and don't really add anything.  Maybe one user with two keyfiles?